src/docs/sphinx/dev_guide/gpu_porting.rst

```diff
@@ -161,6 +160,199 @@ GPU device, or on both a GPU device and a CPU host. For example::
   When Axom is built without RAJA, ``axom::for_all`` becomes a ``for``-loop on
   host (CPU).
 
+%%%%%%%%%%%%%%%%
+Portability
+%%%%%%%%%%%%%%%%
+
+Adherence to the GPU porting guidelines generally results in code that will compile and run on
+multiple backends. However, backends such as CUDA require additional guidelines.
+
+Do not use generic lambda functions (auto parameters) with ``axom::for_all``
```

Comment:
Suggestion: mark "auto" as code, with back-ticks.  This will visually flag `auto` parameters as the point of the portability rule.